solutions/perimeter/fortigate/fortigate-ap-primary.yaml

```diff
@@ -0,0 +1,433 @@
+metadata:
+  name: perimeter-fgt-primary-instance
+  namespace: networking
+  annotations:
+    cnrm.cloud.google.com/project-id: perimeter-project-id # kpt-set: ${perimeter-project-id}
+spec:
+  resourceID: fgt-primary-instance
+  description: Fortigate Primary Instance
+  machineType: n2-standard-4
+  zone: northamerica-northeast1-a
+  canIpForward: true
+  scheduling:
+    automaticRestart: true
+    onHostMaintenance: MIGRATE
+    preemptible: false
+  # AC-2(A) service account is attached to Google Compute Engine
+  serviceAccount:
+    serviceAccountRef:
+      name: perimeter-fortigatesdn-sa
+    scopes:
+    - cloud-platform
+  
+  # TODO: Schedule
+  # resourcePolicies:
+  # - name: TBD
+  
+  # Create a new boot disk from an image
+  bootDisk:
+    autoDelete: true
+    deviceName: persistent-disk-0
+    initializeParams:
+      type: pd-ssd
+      sourceImageRef:
+        external: image-path # kpt-set: ${fgt-primary-image}
+  
+  # Logging Disk from existing resource
+  attachedDisk:
+  - sourceDiskRef:
+      name: perimeter-fgt-primary-log-disk
+  
+  networkInterface:
+  # External
+  - name: nic0
+    networkRef:
+      name: perimeter-global-external-vpc
+    subnetworkRef:
+      name: perimeter-nane1-external-paz-snet
+    networkIpRef:
+      name: perimeter-fgt-primary-ext-address
+      kind: ComputeAddress
+  # Internal
+  - name: nic1
+    networkRef:
+      name: perimeter-global-internal-vpc
+    subnetworkRef:
+      name: perimeter-nane1-internal-paz-snet
+    networkIpRef:
+      name: perimeter-fgt-primary-int-address
+      kind: ComputeAddress
+  # Transit
+  - name: nic2
+    networkRef:
+      name: perimeter-global-transit-vpc
+    subnetworkRef:
+      name: perimeter-nane1-transit-paz-snet
+    networkIpRef:
+      name: perimeter-fgt-primary-transit-address
+      kind: ComputeAddress
+  # MGMT
+  - name: nic3
+    networkRef:
+      name: perimeter-global-mgmt-vpc
+    subnetworkRef:
+      name: perimeter-nane1-mgmt-rz-snet
+    networkIpRef:
+      name: perimeter-fgt-primary-mgmt-address
+      kind: ComputeAddress
+
+  tags:
+  - internet-egress-route
+
+  # Metadata to enable serial console and bootstrap FGT
+  metadata:
+  # AC-17(1) - Serial console is accessible from the gcp console (console.cloud.google.com) and requires a specific set of IAM permissions.
+  - key: serial-port-enable
+    value: "TRUE"
+  - key: license
+    value: LICENSE # kpt-set: ${fgt-primary-license}
+  - key: user-data
+    value: |
+      config system global
+          set hostname "fgt-ap-primary"
+          set pre-login-banner enable
+          set admintimeout 60
+          set timezone 12
+      end
+      config system admin
+          # AC-2(A) - The Fortigates/FortiOS comes with a default local `admin` account.
+          edit "admin"
+              set password fortinet
+          next
+      end
+      config system replacemsg admin "pre_admin-disclaimer-text"
+          set buffer "Acceptable Use Policy
```

Comment:
@CraigBaltzer @amcmullin this is the acceptable policy we discussed the other day